### PR TITLE
Update hosts.example

### PR DESCRIPTION
**Pull request by kborup-redhat**
Added comments for adding prometheus in a offline / disconnected installation, together with node selector.

### CODE REVIEW

File: inventory/hosts.example [hunk 1]
```diff
@@ -752,6 +752,19 @@ debug_level=2
 #
 # Option C - none -- Prometheus, alertmanager and alertbuffer will use emptydir volumes
 # which are destroyed when pods are deleted
+#
+# Deploying Prometheus in a offline / disconnected installation, requires the following addition.
```
Comment (zgalor) [inline inventory/hosts.example]: Why are these required? Those are the defaults for `deployment_type=openshift-enterprise`

Comment (kborup-redhat) [reply inventory/hosts.example]: If not set the setting will be registry.access.redhat.com but if your offline / disconnected it has to be pointed towards your internal registry, where disconnected / offline seriously lack in documentation. 

As far as i can find it does not respond to oreg_url.

Comment (zgalor) [reply inventory/hosts.example]: I'm sorry. I still don't understand.
The params you added to the inventory file are identical to the default ones:
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/vars/openshift-enterprise.yml

So why is that addition required?

Comment (kborup-redhat) [reply inventory/hosts.example]: If you want to install prometheus in a disconnected environment, you need to change those variables to yoru private registry, and as they are not documented anywhere in case you need to change from the default registry.access.redhat.com to lets say docker-registry-default.something.example.com you need to dig trough the playbooks, after you have already run trough them the first time where everything will fail cause you cant reach registry.access.redhat.com

Comment (zgalor) [reply inventory/hosts.example]: Ok. So the new example wasn't clear to me.
Should be something like:
```
# Deploying Prometheus in an offline / disconnected installation requires setting offline image prefix. 
# Each image has its own prefix and version, version can be left out if going with the default Openshift version.
# For example if the offline repo is: docker-registry-default.something.example.com, the following setting should be applied:
# openshift_prometheus_image_prefix=docker-registry-default.something.example.com/openshift3/
```
and all the other images example...

File: inventory/hosts.example [hunk 2]
```diff
@@ -752,6 +752,19 @@ debug_level=2
...
+# Deploying Prometheus in a offline / disconnected installation, requires the following addition. 
+# Each pod in Prometheus have its own image prefix and version, version can be left out if going with the default Openshift version.
+# openshift_prometheus_image_prefix=registry.access.redhat.com/openshift3/
+# openshift_prometheus_image_version=v3.7
+# openshift_prometheus_proxy_image_prefix=registry.access.redhat.com/openshift3/
+# openshift_prometheus_proxy_image_version=v3.7
+# openshift_prometheus_alertbuffer_image_prefix=registry.access.redhat.com/openshift3/
+# openshift_prometheus_alertbuffer_image_version=v3.7
+# openshift_prometheus_alertmanager_image_prefix=registry.access.redhat.com/openshift3/
+# openshift_prometheus_alertmanager_image_version=v3.7
```
Comment (zgalor) [inline inventory/hosts.example]: leave a blank line here

Comment (kborup-redhat) [reply inventory/hosts.example]: Looks good..

**Comment by vrutkovs** [comment]
Master branch is closed! A major refactor is ongoing in devel-40. Changes for 3.x should be made directly to the latest release branch they're relevant to and backported from there.
